_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,89 @@
+---
+title: New API Blueprint tutorial
+excerpt: Learning API Blueprint has never been easier.
```

Comment:
Same as the suggestion above, perhaps, 'With our revamped tutorial, learning the art of API blueprint has never been easier.'